modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2Utils.path

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+<title>CP2Utils</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">CP2Utils</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>MODAL</td>
+	<td>//div[contains(@class,'modal-content')]//p[contains(text(), '${value}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CLOSE_BUTTON</td>
+	<td>//button[contains(@aria-label,'close')]</td>
+	<td></td>
+</tr>
```

Comment:
Did you run source-format for this, Luana? I think it will fail since it is not sorted alphabetically.

Reply:
I ran. I will fix it in the hand.